docs/releases/2.0.txt

```diff
@@ -305,6 +305,13 @@ If you wish to keep this restriction in the admin when editing users, set
     admin.site.unregister(User)
     admin.site.register(User, MyUserAdmin)
 
+Querysets can no longer be reordered after slicing
+--------------------------------------------------
+
+Slicing a queryset before reordering led to unexpected results due to the slice
+being applied only after reordering. Therefore ``reorder`` and ``last`` can no
```

Comment:
Therefore ``reorder`` -> Therefore ``reverse``, since it refers to the `reverse()` method, right?